diff --git a/onnxruntime/test/lora/lora_test.cc b/onnxruntime/test/lora/lora_test.cc
index e8291a3..9a1c376 100644
--- a/onnxruntime/test/lora/lora_test.cc
+++ b/onnxruntime/test/lora/lora_test.cc
@@ -199,39 +199,39 @@ TEST(LoraAdapterTest, Load) {
   }
 }
 
-#ifdef USE_CUDA
-TEST(LoraAdapterTest, VerifyDeviceCopy) {
-  auto cpu_ep = DefaultCpuExecutionProvider();
-  auto cpu_allocator = cpu_ep->CreatePreferredAllocators()[0];
-  auto cuda_ep = DefaultCudaExecutionProvider();
-  auto cuda_allocator = cuda_ep->CreatePreferredAllocators()[0];
-
-  auto gpu_transfer = cuda_ep->GetDataTransfer();
-
-  auto test_params = GenerateTestParameters<float>()();
-  lora::LoraAdapter adapter(std::move(cuda_allocator));
-  adapter.Load(std::move(test_params));
-
-  auto [begin, end] = adapter.GetParamIterators();
-  for (; begin != end; ++begin) {
-    const auto& [_, param] = *begin;
-    const auto& tensor_device = param.GetDeviceOrMapped().Get<Tensor>();
-    ASSERT_EQ(0, strcmp(tensor_device.Location().name, onnxruntime::CUDA));
-
-    const auto& tensor_cpu = param.GetMapped().Get<Tensor>();
-    ASSERT_EQ(tensor_cpu.Shape().Size(), tensor_device.Shape().Size());
-
-    Tensor copy(tensor_cpu.DataType(), tensor_cpu.Shape(), cpu_allocator);
-    ASSERT_TRUE(gpu_transfer->CanCopy(tensor_device.Location().device,
-                                      copy.Location().device));
-    ASSERT_STATUS_OK(gpu_transfer->CopyTensor(tensor_device, copy));
-
-    auto expected_span = tensor_cpu.DataAsSpan<float>();
-    auto copy_span = copy.DataAsSpan<float>();
-
-    ASSERT_EQ(expected_span, copy_span);
-  }
-}
-#endif
+// #ifdef USE_CUDA
+// TEST(LoraAdapterTest, VerifyDeviceCopy) {
+//   auto cpu_ep = DefaultCpuExecutionProvider();
+//   auto cpu_allocator = cpu_ep->CreatePreferredAllocators()[0];
+//   auto cuda_ep = DefaultCudaExecutionProvider();
+//   auto cuda_allocator = cuda_ep->CreatePreferredAllocators()[0];
+
+//   auto gpu_transfer = cuda_ep->GetDataTransfer();
+
+//   auto test_params = GenerateTestParameters<float>()();
+//   lora::LoraAdapter adapter(std::move(cuda_allocator));
+//   adapter.Load(std::move(test_params));
+
+//   auto [begin, end] = adapter.GetParamIterators();
+//   for (; begin != end; ++begin) {
+//     const auto& [_, param] = *begin;
+//     const auto& tensor_device = param.GetDeviceOrMapped().Get<Tensor>();
+//     ASSERT_EQ(0, strcmp(tensor_device.Location().name, onnxruntime::CUDA));
+
+//     const auto& tensor_cpu = param.GetMapped().Get<Tensor>();
+//     ASSERT_EQ(tensor_cpu.Shape().Size(), tensor_device.Shape().Size());
+
+//     Tensor copy(tensor_cpu.DataType(), tensor_cpu.Shape(), cpu_allocator);
+//     ASSERT_TRUE(gpu_transfer->CanCopy(tensor_device.Location().device,
+//                                       copy.Location().device));
+//     ASSERT_STATUS_OK(gpu_transfer->CopyTensor(tensor_device, copy));
+
+//     auto expected_span = tensor_cpu.DataAsSpan<float>();
+//     auto copy_span = copy.DataAsSpan<float>();
+
+//     ASSERT_EQ(expected_span, copy_span);
+//   }
+// }
+// #endif
 }  // namespace test
 }  // namespace onnxruntime
