diff --git a/onnxruntime/test/providers/cuda/nhwc/nhwc_cuda_helper.h b/onnxruntime/test/providers/cuda/nhwc/nhwc_cuda_helper.h
index 1aea58c..d55bb34 100644
--- a/onnxruntime/test/providers/cuda/nhwc/nhwc_cuda_helper.h
+++ b/onnxruntime/test/providers/cuda/nhwc/nhwc_cuda_helper.h
@@ -58,7 +58,7 @@ namespace test {
 template <typename T>
 class CudaNhwcTypedTest : public ::testing::Test {};
 
-using CudaNhwcTestTypes = ::testing::Types<float, MLFloat16>;  // double,
+using CudaNhwcTestTypes = ::testing::Types<>;  // double,
 TYPED_TEST_SUITE(CudaNhwcTypedTest, CudaNhwcTestTypes);
 }  // namespace test
 }  // namespace onnxruntime
