diff --git a/onnxruntime/test/shared_lib/test_inference.cc b/onnxruntime/test/shared_lib/test_inference.cc
index 0a4ea71..2eb9cd4 100644
--- a/onnxruntime/test/shared_lib/test_inference.cc
+++ b/onnxruntime/test/shared_lib/test_inference.cc
@@ -578,41 +578,41 @@ TEST(CApiTest, SparseInputModel) {
 #endif  // !defined(DISABLE_SPARSE_TENSORS)
 
 // Memory leak
-#ifndef ABSL_HAVE_ADDRESS_SANITIZER
-TEST(CApiTest, custom_op_handler) {
-  std::cout << "Running custom op inference" << std::endl;
-
-  std::vector<Input<float>> inputs(1);
-  auto& input = inputs[0];
-  input.name = "X";
-  input.dims = {3, 2};
-  input.values = {1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f};
-
-  // prepare expected inputs and outputs
-  std::vector<int64_t> expected_dims_y = {3, 2};
-  std::vector<float> expected_values_y = {2.0f, 4.0f, 6.0f, 8.0f, 10.0f, 12.0f};
-
-#ifdef USE_CUDA
-  cudaStream_t compute_stream = nullptr;
-  cudaStreamCreateWithFlags(&compute_stream, cudaStreamNonBlocking);
-  MyCustomOp custom_op{onnxruntime::kCudaExecutionProvider};
-#else
-  MyCustomOp custom_op{onnxruntime::kCpuExecutionProvider};
-#endif
-
-  Ort::CustomOpDomain custom_op_domain("test");
-  custom_op_domain.Add(&custom_op);
-
-#ifdef USE_CUDA
-  TestInference<float>(*ort_env, CUSTOM_OP_MODEL_URI, inputs, "Y", expected_dims_y, expected_values_y, 1,
-                       custom_op_domain, nullptr, false, compute_stream);
-  cudaStreamDestroy(compute_stream);
-#else
-  TestInference<float>(*ort_env, CUSTOM_OP_MODEL_URI, inputs, "Y", expected_dims_y, expected_values_y, 0,
-                       custom_op_domain, nullptr);
-#endif
-}
-#endif
+// #ifndef ABSL_HAVE_ADDRESS_SANITIZER
+// TEST(CApiTest, custom_op_handler) {
+//   std::cout << "Running custom op inference" << std::endl;
+
+//   std::vector<Input<float>> inputs(1);
+//   auto& input = inputs[0];
+//   input.name = "X";
+//   input.dims = {3, 2};
+//   input.values = {1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f};
+
+//   // prepare expected inputs and outputs
+//   std::vector<int64_t> expected_dims_y = {3, 2};
+//   std::vector<float> expected_values_y = {2.0f, 4.0f, 6.0f, 8.0f, 10.0f, 12.0f};
+
+// #ifdef USE_CUDA
+//   cudaStream_t compute_stream = nullptr;
+//   cudaStreamCreateWithFlags(&compute_stream, cudaStreamNonBlocking);
+//   MyCustomOp custom_op{onnxruntime::kCudaExecutionProvider};
+// #else
+//   MyCustomOp custom_op{onnxruntime::kCpuExecutionProvider};
+// #endif
+
+//   Ort::CustomOpDomain custom_op_domain("test");
+//   custom_op_domain.Add(&custom_op);
+
+// #ifdef USE_CUDA
+//   TestInference<float>(*ort_env, CUSTOM_OP_MODEL_URI, inputs, "Y", expected_dims_y, expected_values_y, 1,
+//                        custom_op_domain, nullptr, false, compute_stream);
+//   cudaStreamDestroy(compute_stream);
+// #else
+//   TestInference<float>(*ort_env, CUSTOM_OP_MODEL_URI, inputs, "Y", expected_dims_y, expected_values_y, 0,
+//                        custom_op_domain, nullptr);
+// #endif
+// }
+// #endif
 
 #ifdef USE_CUDA
 TEST(CApiTest, custom_op_set_input_memory_type) {
