diff --git a/cmake/onnxruntime.cmake b/cmake/onnxruntime.cmake
index 6c1d448..4ea2943 100644
--- a/cmake/onnxruntime.cmake
+++ b/cmake/onnxruntime.cmake
@@ -80,6 +80,9 @@ if(WIN32)
     "${ONNXRUNTIME_ROOT}/core/dll/delay_load_hook.cc"
     "${ONNXRUNTIME_ROOT}/core/dll/onnxruntime.rc"
   )
+
+  # Workaround for https://github.com/conda-forge/onnxruntime-feedstock/pull/56#issuecomment-1586080419
+  set_target_properties(onnxruntime PROPERTIES OUTPUT_NAME onnxruntime_conda)
 elseif(onnxruntime_BUILD_APPLE_FRAMEWORK)
   # apple framework requires the header file be part of the library
   onnxruntime_add_shared_library(onnxruntime
